commands/shooter/shootsimnote.py

```diff
@@ -0,0 +1,23 @@
+from commands2 import Command
+from subsystems.shootersubsystem import ShooterSubsystem
+
+# temp command for testing note sim, general score command will be made later
+
+
+class ShootSimNote(Command):
```

Comment:
merge this with the feed note, triggering simulation should be the same as note feed